crypto/evp/p_lib.c

```diff
@@ -237,6 +237,17 @@ void *EVP_PKEY_get0(const EVP_PKEY *pkey)
     return pkey->pkey.ptr;
 }
 
+const unsigned char *EVP_PKEY_get_hmac(const EVP_PKEY *pkey, int *len)
```

Comment:
I think this falls in the "missing accessors now that stuff is opaque" category, so if it's really needed, yes it should make it to 1.1.0.